readme.md

```diff
@@ -50,14 +52,23 @@ You can control the maven build with the following profiles:
 * macm1
 
 ### Examples
-    mvn clean verify -Pwin64,latest,NOSDK -DskipTests=true (builds for latest eclipse and windows bits)
-    mvn clean verify -Plinux32,latest.NOSDK -DskipTests=true (builds for latest eclipse and linux 32 bits)
-    mvn clean verify -PSDK,win64,latest -DskipTests=true (builds the Sloeber SDK. For Sloeber programmers.)
-    
-To build for latest and the platform you are running on:
-
-    mvn clean verify -DskipTests=true
 
+* Build the latest version for the platform you are running on:
+
+    `mvn clean verify -DskipTests=true`
+
+* Build Eclipse + Sloeber for 64-bit Windows:
+    `mvn clean verify -Pwin64,latest,NOSDK -DskipTests=true`
+		
+* Build Eclipse + Sloeber for 64-bit Linux:
+    `mvn clean verify -Plinux64,latest,NOSDK -DskipTests=true`
+		
+* Build Eclipse + Sloeber for 32-bit Linux:
+    `mvn clean verify -Plinux32,latest,NOSDK -DskipTests=true`
```

Comment:
Sure thing.  I saw the website pages are also in this repository so I'll update the Linux download `chmod` note as well (can't test that the download page handles `tar.gz` correctly though since I don't have PHP installed).  I'll add a commit with all the additional stuff.

Reply:
Should 32-bit references be removed for all platforms?

Reply:
You can leave download links on the web pages to me. The download links to the web pages are based on files and only updated when I make an official release.